packages/hub/src/relative-values/components/RelativeValuesDefinitionForm/index.tsx

```diff
@@ -26,9 +21,29 @@ import {
   TrashIcon,
 } from "@quri/ui";
 
-import { Header } from "@/components/ui/Header";
 import { SelectCluster } from "../SelectCluster";
 
+const formSectionHeader = (text: string) => (
+  <div className="text-base font-semibold leading-7 text-gray-900">{text}</div>
+);
```

Comment:
I'm curious why you sometimes use functions instead of real React components. For brevity, to avoid naming props, I guess?

Also, I know I said semantic HTML is dead in another comment, but I'd still use `<header>` here instead of `<div>`.

Reply:
Brevity, that's all